src/task7/task7

```diff
@@ -0,0 +1,34 @@
+function arrayToList(arr) {
+    let list = {};
+    for (let i = arr.length - 1; i >= 0; i--) {
+        list.value = arr[i];
+        list.rest = arr.length > 1 ? arrayToList(arr.slice(1)) : null;
+    }
+    return list;
+}
+
+function listToArray(list) {
+    let array = [];
+    for (; list; list = list.rest) {
+        array.push(list.value);
+    }
+    return array;
+}
+
+function prepend(list, value) {
+    return list = { value: value, rest: list };
```

Comment:
1. assignment in the return? why is that even work?
2. **value: value** dont like this!

Reply:
haha
no idea myself tbh